outline.md

```diff
@@ -34,50 +34,48 @@ and how tasks get assigned and executed on the cluster.
 Learners will understand how nodes, CPUs, memory, etc., are organized,
 and "how it looks like one remote computer when it has a bazillion CPUs and several Terabytes?".
 At the end of the section, they will
-submit a batch job, trace its execution, and examine its results.
+submit a number of batch jobs for serial, parallel and task array workloads , trace their execution, and examine their results.
 Throughout, it will be stressed that different HPC sites
 have different setups, schedulers, policies and resources,
-and how learners can translate the concepts accordingly.
+and how learners can translate the concepts accordingly. The conepts we present will allow learners to quickly apply the techniques to their local clusters.
 
 ### Parallel workflows
 
 In this section, learners will be presented with a specific research scenario,
 for example:
 
-> Lola was hired by a research lab to help prepare the purchase of a multi-million experiment.
+> Lola was hired by a research lab to help prepare the purchase of a multi-million dollar experiment.
 The experiment is known to fail at temperatures that are too low or too high.
-So Lola's task is to research the temperatures at a certain section of the lab room, where the device
-will be placed.
+So Lola's task is to research the temperatures in a certain section of the lab where the device will be placed.
 She knows that the temperature changes follow a daily pattern,
 and she's written some code to simulate these temperature changes.
 After running this code and generating the temperature predictions,
-she determine how closely her predictions match the actual temperature readings
+she determines how closely her predictions match the actual temperature readings
 she has for every day in the last year.
 The simuation would take too long to run
 and generate too much data for her lab workstation,
 so she will use the local University's HPC facility for this work.
 
-They will then walk through the different steps involved, including
-data transfer,
-software installation and
-developing batch scripts to submit jobs for
-performing the simulation and analyzing the results.
+They will then guide learners through the different steps involved, including
```

Comment:
"They" here refers to "the learners" - so maybe "they will then be guided through"? Although I think this makes it sound a bit passive on the learners' part.